scoped_model_counter/test/widget_test.dart

```diff
@@ -1,14 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:scoped_model/scoped_model.dart';
+import 'package:provider/provider.dart';
 import 'package:scoped_model_counter/main.dart';
 
 void main() {
   testWidgets('Counter increments smoke test', (WidgetTester tester) async {
     // Build our app, provide it with a model, and trigger a frame.
     await tester.pumpWidget(
-      ScopedModel(
-        model: Counter(),
+      ChangeNotifierProvider(
+        builder: (_) => Counter(),
```

Comment:
nit: There are some other spots in here where the underscore for "param I don't care about" could be used but isn't (line 67 in `main.dart` for example). I personally prefer to avoid underscores, but really all that matters is that the code picks one way and sticks with it.